README.md

```
@@ -1,3 +1,241 @@
+docker compose up -d
+```
+
+### Updating
+
+Update scripts for Linux (`update.sh`) and Windows (`update.bat`) are included in the repository. The Docker image is automatically built and published to `ghcr.io/tanjunbot/new_tanjun:latest` on pushes to master.
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)